addons/account/models/account_payment.py

```diff
@@ -366,9 +366,13 @@ def open_payment_matching_screen(self):
     def _compute_destination_account_id(self):
         for payment in self:
             if payment.invoice_ids:
-                payment.destination_account_id = payment.invoice_ids[0].mapped(
+                destination_account_id = payment.invoice_ids[0].mapped(
```

Comment:
Nitpicking: there might be more than 1 account in your variable, I would call it ```destination_account_ids```